lib/Twig/Environment.php

```diff
@@ -641,7 +641,7 @@ public function compileSource($source, $name = null)
             $compiled = $this->compile($this->parse($this->tokenize($source, $name)), $source);
 
             if (isset($source[0])) {
-                $compiled .= '/* '.str_replace(array("\r\n", "\r", "\n"), "*/\n/* ", str_replace('*/', '*//*', $source))."*/\n";
+                $compiled .= '/* '.strtr($source, array('*/' => '*//*', "\r\n" => "*/\n/* ", "\r" => "*/\n/* ", "\n" => "*/\n/* "))."*/\n";
```

Comment:
Why not keep CRLF in the code? You replace with with LF only. This changes the source code line endings. So it does not actually return the real source code.